monitors-driver-library/src/main/java/cloud/benchflow/monitors/MonitorFactory.java

```diff
@@ -0,0 +1,69 @@
+package cloud.benchflow.monitors;
+
+import cloud.benchflow.monitors.cpu.CpuMonitor;
+import cloud.benchflow.monitors.querymysql.QueryMySQLMonitor;
+
+import com.sun.faban.driver.transport.hc3.ApacheHC3Transport;
+
+import java.util.Map;
+import java.util.logging.Logger;
+
+/**
+ * @author Simone D'Avico (simonedavico@gmail.com)
+ *
+ * Created on 13/07/16.
+ */
+public class MonitorFactory {
+
+    public static Monitor getMonitor(ApacheHC3Transport http,
+                                     String monitorName,
+                                     Map<String, String> parameters,
+                                     String endpoint,
+                                     String startApi,
+                                     String stopApi,
+                                     String monitorApi) {
+
+        MonitorAPI api = new MonitorAPI(startApi, monitorApi, stopApi);
+        switch (monitorName) {
```

Comment:
Try to use reflection instead, so that we don't need to change the Factory when we add new monitors.